client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -0,0 +1,199 @@
+	> div {
+		width: 460px;
+		padding: 0 35px;
+		> div {
+			min-width: 144px;
+		}
+	}
+
+	div {
+		min-width: 144px;
+	}
+`;
+
+const MarketplaceNextSteps = styled.div< MarketplaceThemeProps >`
+	h3 {
+		font-weight: ${ ( { theme } ) => theme?.weights.bold };
+	}
+
+	p {
+		color: var( --studio-gray-50 );
+		padding-right: 20px;
+	}
+	> div {
+		display: flex;
+	}
+`;
+
+const MarketplaceThankYou = () => {
+	const selectedSite = useSelector( getSelectedSite );
+	const previousPath = useSelector( getPreviousPath );
+	const menuItems = useSiteMenuItems();
+
+	const { url: postsPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'edit-php' ) ?? {};
+
+	const { url: yoastSeoPageUrl } =
+		menuItems.find( ( { slug }: { slug: string } ) => slug === 'wpseo_dashboard' ) ?? {};
+
+	const translate = useTranslate();
+	const imageWidth = isDesktop() ? 200 : 150;
+
+	return (
+		<>
+			<Masterbar>
+				<Item
+					icon="cross"
+					onClick={ () =>
+						previousPath
+							? page( previousPath )
+							: page(
+									`/plugins/wordpress-seo${ selectedSite?.slug ? `/${ selectedSite.slug }` : '' }`
+							  )
+					}
+					tooltip={ translate( 'Go to plugin' ) }
+					tipTarget="close"
+				/>
+			</Masterbar>
+			<MarketplaceThankYouContainer className="marketplace-thank-you__container">
+				<MarketplaceThankYouHeader>
+					<img alt="yoast logo" width={ imageWidth } src={ yoastInstalledImage }></img>
+				</MarketplaceThankYouHeader>
+				<ThankYouBody>
+					<div>
+						<MarketplaceThankyouSection>
+							<MarketplaceHeaderTitle className="marketplace-thank-you__body-header wp-brand-font">
+								{ translate( 'Yoast SEO Premium is installed' ) }
+							</MarketplaceHeaderTitle>
+							<p>{ translate( "You're all set. Yoast SEO is installed and ready to go." ) }</p>
+						</MarketplaceThankyouSection>
+						<MarketplaceThankyouSection>
+							<MarketplaceHeaderTitle
+								subtitle
+								className="marketplace-thank-you__body-header wp-brand-font"
+							>
+								{ translate( 'What’s next?' ) }
+							</MarketplaceHeaderTitle>
+							<MarketplaceNextSteps>
+								<h3>{ translate( 'Plugin setup' ) }</h3>
+								<div>
+									<p>
+										{ translate(
+											'Get to know Yoast SEO and customize it, so you can hit the ground running.'
+										) }
+									</p>
+									<div>
+										<FullWidthButton
+											onClick={ () => ( window.location.href = yoastSeoPageUrl ) }
+											primary
+										>
+											{ translate( 'Get started' ) }
+										</FullWidthButton>
+									</div>
+								</div>
+								<h3>{ translate( 'Start putting SEO to work' ) }</h3>
+								<div>
+									<p>
+										{ translate(
+											"Improve your site's performance and rank higher with a few tips."
+										) }
+									</p>
+									<div>
+										<FullWidthButton onClick={ () => ( window.location.href = postsPageUrl ) }>
+											{ translate( 'View posts' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **121** times:
`translate( 'View Posts' )` *ES Score: 8*